include/votca/xtp/aobasis.h

```diff
@@ -104,7 +108,7 @@ class AOBasis
     
     int getMaxFunctions ( );
     
-private:
+  
     unsigned int _AOBasisSize;
```

Comment:
unsigned or unsigned int?

Reply:
is the same thing, ```unsigned``` is short for ```unsigned int```

Reply:
I know, but some lines above you replace unsigned int by unsigned, so I wondered about consistency.

Reply:
yeah in the end I want to completely move to ```unsigned```